src/backend/marsha/settings.py

```diff
@@ -154,6 +154,28 @@ class Base(Configuration):
     AUTH_USER_MODEL = "core.User"
 
     ASGI_APPLICATION = "marsha.asgi.application"
+    """
+        For sentinels:
+        CHANNEL_LAYERS = {
+        'default': {
+            'BACKEND': 'channels_redis.core.RedisChannelLayer',
+            'CONFIG': {
+                "hosts": [{
+                    "sentinels": [(SENTINEL_HOST, SENTINEL_PORT)]m
+                    "master_name": SENTINEL_MASTER,
+                }],
+            },
+        },
+    }
+    """
```

Comment:
Maybe use # on each line if this comment is to stay? 
So does it work with sentinels only or redis cluster as well?

Reply:
sentinels only. Redis Cluster is in WIP https://aioredis.readthedocs.io/en/v1.1.0/index.html